access/handler.go

```diff
@@ -35,6 +35,11 @@ func (h *Handler) Ping(ctx context.Context, _ *access.PingRequest) (*access.Ping
 	return &access.PingResponse{}, nil
 }
 
+func (h *Handler) GetLatestProtocolStateSnapshot(_ context.Context,
+	_ *access.GetLatestProtocolStateSnapshotRequest) (*access.ProtocolStateSnapshotResponse, error) {
+	panic("not yet implemented")
```

Comment:
Maybe just return an error that this function is not implemented yet?

Someone might call this function and that would bring down the whole access node.

Reply:
yup ..my bad